website/templates/project/files.mako

```diff
@@ -29,8 +29,11 @@ ${parent.javascript_bottom()}
 <script src=${"/static/public/js/files-page.js" | webpack_asset}></script>
 <script type="text/javascript">
     window.contextVars = window.contextVars || {};
-    % if 'write' in user['permissions'] and not node['is_registration'] and not disk_saving_mode:
-        window.contextVars.uploadInstruction = true
+
+    <% import json %>
+    % if 'write' in user['permissions'] and not node['is_registration']:
+        window.contextVars.uploadInstruction = !${json.dumps(disk_saving_mode)};
```

Comment:
This variable name is misleading; I expected `uploadInstruction` to be a string, not a boolean. Why not name it `diskSavingMode`?